test/12_conditional_commit.ts

```diff
@@ -447,280 +602,295 @@ describe('Create Voucher sets and commit to vouchers with token conditional comm
+            );
+            assert.equal(
+              promiseData[constants.PROMISE_DATA_FIELDS.validFrom].toString(),
+              constants.PROMISE_VALID_FROM.toString(),
+              'Promise data field -> validFrom is incorrect'
+            );
+
+            assert.equal(
+              promiseData[constants.PROMISE_DATA_FIELDS.validTo].toString(),
+              constants.PROMISE_VALID_TO.toString(),
+              'Promise data field -> validTo is incorrect'
+            );
+            assert.equal(
+              promiseData[constants.PROMISE_DATA_FIELDS.idx].toString(),
+              constants.ZERO.toString(),
+              'Promise data field -> idx is incorrect'
+            );
+
+            const promiseSeller = await contractVoucherKernel.getSupplyHolder(
+              tokenSupplyKey
+            );
+
+            assert.strictEqual(
+              promiseSeller,
+              users.seller.address,
+              'Seller incorrect'
+            );
+
+            const promiseOrderData = await contractVoucherKernel.getOrderCosts(
+              tokenSupplyKey
+            );
+            assert.isTrue(
+              promiseOrderData[constants.PROMISE_ORDER_FIELDS.price].eq(
+                BN(constants.PROMISE_PRICE1)
+              ),
+              'Promise produt price mismatch'
+            );
+            assert.isTrue(
+              promiseOrderData[constants.PROMISE_ORDER_FIELDS.depositSe].eq(
+                BN(constants.PROMISE_DEPOSITSE1)
+              ),
+              'Promise seller deposit mismatch'
+            );
+            assert.isTrue(
+              promiseOrderData[constants.PROMISE_ORDER_FIELDS.depositBu].eq(
+                BN(constants.PROMISE_DEPOSITBU1)
+              ),
+              'Promise buyer deposit mismatch'
+            );
+
+            const tokenNonce = await contractVoucherKernel.getTokenNonce(
+              users.seller.address
+            );
+            assert.isTrue(
+              tokenNonce.eq(constants.ONE),
+              'Voucher kernel nonce mismatch'
+            );
+
+            assert.equal(
+              promiseId,
+              await contractVoucherKernel.getPromiseIdFromSupplyId(
+                tokenSupplyKey
+              ),
+              'PromisId mismatch'
+            );
+
+            // Check VoucherSets state
+            const sellerVoucherSetsBalance = (
+              await contractVoucherSets.functions[fnSignatures.balanceOf1155](
+                users.seller.address,
+                tokenSupplyKey
+              )
+            )[0];
+
+            assert.isTrue(
+              sellerVoucherSetsBalance.eq(constants.QTY_10),
+              'VoucherSets seller balance mismatch'
+            );
+          });
+
+          describe('Flow with automatic gate.registerVoucherSetId', () => {
+            it('Should be able to create Voucher with gate address and non empty nft token id', async () => {
```

Comment:
Change to conditional token Id

Reply:
Replaced "nft token id" with "conditional token id" in several locations throughout the contract.